src/assets/css/_sass/components/_home-made-local.scss

```diff
@@ -0,0 +1,38 @@
+.made-local-section {
+  @include font-body;
+  font-weight: 100;
+  color: $color-text--dark;
+  padding-top: 120px;
+  height: 1200px;
+
+  .section__header {
+    text-align: center;
+    width: 600px;
+    @include margin-center;
+  }
+
+  .section__title {
+    @include h3;
+    @include font-display;
+    text-transform: uppercase;
+  }
+
+  .section__lead {
+    width: 570px;
+    line-height: 30px;
+    margin: 20px auto;
```

Comment:
on line 10 if you make `.section__header` with `570px` you can remove the width and margin properties here.